app/assets/javascripts/application.js

```diff
@@ -33,7 +33,33 @@ $(function() {
 });
 
 function executeAttachinary() {
+  var js_template = [
+    "<ul class='attachinary_images_list'>",
```

Comment:
@akitaonrails looks like attachinary use its own weird "template engine" https://github.com/assembler/attachinary/blob/master/lib/assets/javascripts/attachinary.js.coffee#L183. Maybe we could drop it there and something like `JST['attachinary_template'].raw()`, but I'm not sure. I'm investigating it right now.